scripts/build-php.sh

```diff
@@ -44,8 +61,8 @@ if [ -n "$PHP_ASC_URL" ]; then
 fi
 tar Jxvf php.tar.xz
 rm php.tar.xz.asc php.tar.xz
-
-cd /usr/src/php-$PHP_VERSION/
+mv php-$PHP_VERSION php
```

Comment:
Move the source to a directory that does not include the exact version number since we need to be able to locate this in the next stage of the Dockerfile.